run-example.sh

```diff
@@ -12,46 +12,52 @@ keepContainers=${keepContainers:-false}
 # Defaults for selected dotnet CLI commands.
 configuration=${configuration:-Release}
 aspNetAppTargetFramework=${aspNetAppTargetFramework:-netcoreapp3.1}
-sampleAppTargetFramework=${sampleAppTargetFramework:-netcoreapp3.1}
-sampleApp=${sampleApp:-ConsoleApp}
+exampleAppTargetFramework=${exampleAppTargetFramework:-netcoreapp3.1}
+exampleApp=${exampleApp:-ConsoleApp}
+
+# Handle the differences between launching a dll and exe
+exampleAppExt="dll"
+exampleAppDotnetCli="dotnet"
+if [[ $exampleAppTargetFramework == net4* ]]; then
+  exampleAppExt="exe"
+  exampleAppDotnetCli=""
+fi
 
 # Defaults for selected OTEL env vars.
 exporter=${exporter:-otlp}
-sampleAppInjectSDK=${sampleAppInjectSDK:-true}
+exampleAppInjectSDK=${exampleAppInjectSDK:-true}
 
 # Build the applications
 if [[ $skipAppBuild != "true" && $skipAppBuild != "1" ]]; then
   # Build the server app
-  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./samples/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj
+  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./examples/AspNetCoreMvc/Examples.AspNetCoreMvc.csproj
 
   # build plugin for HTTP server app
-  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./samples/Vendor.Distro/Vendor.Distro.csproj -o bin/tracer-home/$aspNetAppTargetFramework
+  dotnet publish -f $aspNetAppTargetFramework -c $configuration ./examples/Vendor.Distro/Examples.Vendor.Distro.csproj -o bin/tracer-home/$aspNetAppTargetFramework
 
   # build the client app
-  dotnet publish -f $sampleAppTargetFramework -c $configuration ./samples/${sampleApp}/${sampleApp}.csproj
+  dotnet publish -f $exampleAppTargetFramework -c $configuration ./examples/${exampleApp}/Examples.${exampleApp}.csproj
 fi
 
 function finish {
   if [[ $keepContainers != "true" && $keepContainers != "1" ]]; then
-    docker-compose -f ./samples/docker-compose.yaml down
-    docker-compose -f ./dev/docker-compose.yaml down
+    docker-compose -f ./dev/docker-compose.yaml -f ./examples/docker-compose.yaml down
```

Comment:
@pellared as you expected: only one docker network if we use a single call with 2 scripts